packages/note/src/css/index.js

```diff
@@ -0,0 +1,129 @@
+import core from '@pluralsight/ps-design-system-core'
+import { names as themeNames } from '@pluralsight/ps-design-system-theme/vars'
+
+export const resetButton = {
+  background: 'transparent',
+  border: 'none',
+  color: 'inherit',
+  font: 'inherit',
+  lineHeight: 'normal',
+  margin: 0,
+  overflow: 'visible',
+  padding: 0,
+  width: 'auto',
+
+  MozOsxFontSmoothing: 'inherit',
+  WebkitAppearance: 'none',
+  WebkitFontSmoothing: 'inherit',
+
+  '&::-moz-focus-inner': {
+    border: 0,
+    padding: 0
+  }
+}
+
+export default {
+  '.psds-note__list': {
+    listStyle: 'none',
+    marginLeft: 0
+  },
+
+  '.psds-note': {
+    alignItems: 'flex-start',
+    display: 'flex',
+    fontWeight: core.type.fontWeightMedium,
+    lineHeight: core.type.lineHeightStandard,
+    padding: `${core.layout.spacingLarge} ${core.layout.spacingMedium}`,
+
+    'li &': { borderTop: `1px solid ${core.colors.gray03}` },
```

Comment:
I think this is a first for... goodness, what is this kind of selector called...  the "when I'm a child of a particular parent" selector :)... anyway, we have traditionally tried to be very limited on nesting that we were willing to do.  Is this new type of nesting one we want to put in-bounds generally?  Or in the particular case, do we want to make an exception?  

The alternative is something like a top-level `.psds-note__list-item .psds-note`, right?